third_party/BUILD

```diff
@@ -499,7 +499,10 @@ distrib_java_import(
 # For bootstrapping JavaBuilder
 distrib_jar_filegroup(
     name = "guava-jars",
-    srcs = ["guava/guava-31.0.1-jre.jar"],
+    srcs = [
+      "guava/failureaccess-1.0.1.jar",
```

Comment:
You probably need to split this into two filegroups
```
filegroup(
    name = "guava-jar",
    srcs = ["guava/guava-31.1-jre.jar"],
)

filegroup(
    name = "guava-failureaccess-jar",
    srcs = ["guava/guava-31.1-jre.jar"],
)

distrib_jar_filegroup(
    name = "guava-jars",
    srcs = [
        ":guava-jar",
        ":guava-failureaccess-jar",
    ],
    enable_distributions = ["debian"],
)
```
Then you can add them separately above:
```
"--classpath_entry $(location //third_party:guava-jar) " +
"--classpath_entry $(location //third_party:guava-failureaccess-jar) " +
```

Reply:
Experimentally, it seems like it's not necessary to do literally that. `6fbe89e` passes CI. Is that commit fine? And do I really need to have used `distrib_jar_filegroup` for the new `//third_party:guava-failureaccess-jar`?

Reply:
Also, if `srcs` in `:guava-jars` has more than one element then we run into the issue mentioned in `b98be959`.

Reply:
I see, it looks good. Using `distrib_jar_filegroup` is for the Bazel Debian build to switch to system installed libraries easily (eg. `/usr/share/java/guava.jar`), it's better to keep this pattern in case we want to package a new Bazel for Debian.